src/main/java/com/autotune/analyzer/loop/EMInterface.java

```diff
@@ -1,75 +1,75 @@
 package com.autotune.analyzer.loop;
 
 import com.autotune.analyzer.AutotuneExperiment;
-import com.autotune.common.k8sObjects.Metric;
 import com.autotune.common.experiments.ExperimentTrial;
-import com.autotune.common.experiments.TrialDetails;
+import com.autotune.common.k8sObjects.Metric;
 import com.autotune.utils.HttpUtils;
 import com.autotune.utils.TrialHelpers;
-import org.json.JSONObject;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.net.URL;
 import java.sql.Timestamp;
 import java.time.Instant;
 
-import static com.autotune.utils.AnalyzerConstants.ServiceConstants.*;
-import static com.autotune.utils.AutotuneConstants.JSONKeys.*;
+import static com.autotune.utils.AutotuneConstants.JSONKeys.REQUEST_COUNT;
+import static com.autotune.utils.AutotuneConstants.JSONKeys.REQUEST_SUM;
 import static com.autotune.utils.ExperimentMessages.RunExperiment.*;
 import static com.autotune.utils.ServerContext.EXPERIMENT_MANAGER_CREATE_TRIAL_END_POINT;
 
 /**
  *
  */
 public class EMInterface {
-	private static final Logger LOGGER = LoggerFactory.getLogger(HPOInterface.class);
-
-	/**
-	 *
-	 * @param autotuneExperiment
-	 * @param experimentTrial
-	 */
-	public static void SendTrialToEM(AutotuneExperiment autotuneExperiment,
-											  ExperimentTrial experimentTrial) {
-		try {
-			int trialNumber = experimentTrial.getTrialInfo().getTrialNum();
-			// Prepare to send the trial config to EM
-			String experimentTrialJSON = TrialHelpers.experimentTrialToJSON(experimentTrial);
+    private static final Logger LOGGER = LoggerFactory.getLogger(HPOInterface.class);
```

Comment:
Can `HPOInterface.class` be changed to `EMInterface.class`?

Reply:
done